README.md

```diff
@@ -8,9 +8,20 @@ Derelict|Prisoner
 -|-
 ![](media/carousel.png) | ![](media/prisoner.png)
 
+# Build guide
+```
+$ git clone https://github.com/Wunkolo/vkblam && cd vkblam
+$ git clone https://github.com/vector-of-bool/cmrc external/cmrc
+$ git clone https://github.com/mandreyel/mio external/mio
+$ git clone https://github.com/g-truc/glm external/glm
+$ mkdir _build && cd _build
+$ cmake ..
+$ make -j$(nproc) 
```

Comment:
You shouldn't assume the user's build system to be `make`. CMake provides a way to build using whatever the user configured as their default build system
A better process would be:
```
mkdir build
cd build
cmake ..
cmake --build .
```

Reply:
Good point, personally just never thought of that. Ill push that as a patch to my fork then resolve this conversation.

Reply:
Done as of commit [ce2b26b](https://github.com/Wunkolo/vkblam/pull/1/commits/ce2b26beb019a45767eaa2389a6fee01cc390b2d)